### DISCUSSION TITLE
Dev Tools opens in new window instead of a tab

### DISCUSSION THREAD

**Discussion by danley0** [community/community#76742] [Programming Help] [answered]
### Body

I apologise if this is in the wrong category, I am new to GitHub. Recently I was exploring VSCode, and unknowingly reset a parameter. Previously when I ran a javascript file in Live Server in VSCode, a chrome window would open. I would right-click and select "inspect" to go to the DevTools view, then Console or Sources. The DevTools would open in the previous window where I selected "inspect", as a new tab. NOW it opens in a completely new window. How can I make DevTools open as a tab in the chrome "inspect" window? Any help in Chrome is outdated and doesn't work. I have checked all the help and even emailed Google Chrome Help, without a reply. HOpe this makes sense. Attached screenshots explain.

![Screenshot 2023-11-23 at 17 17 16](https://github.com/community/community/assets/63583032/d914d07b-aabd-4b89-a007-541a0a3872be)
![Screenshot 2023-11-23 at 17 22 52](https://github.com/community/community/assets/63583032/16507dbd-6ab3-4846-b423-db966b17649b)
![Screenshot 2023-11-23 at 17 20 20](https://github.com/community/community/assets/63583032/78ae5d72-f2c0-46ca-9434-27af743fb109)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ihnatenkoo** [comment]
Hello, this will help you set up your dev tools
![Screenshot_1](https://github.com/community/community/assets/89586345/b5e03fe9-b502-4878-967d-2cd0444c2f89)

**Reply by leeringrose** [reply, marked as answer]
This would work fine!
@danley0  Have you got yours fixed? If so I think you should put the `answered` tag to this question.
